builder/lxc/communicator.go

```diff
@@ -78,14 +81,14 @@ func (c *LxcAttachCommunicator) Upload(dst string, r io.Reader, fi *os.FileInfo)
 		// rename tempfile to match original file name. This makes sure that if file is being
 		// moved into a directory, the filename is preserved instead of a temp name.
 		adjustedTempName := filepath.Join(tfDir, (*fi).Name())
-		mvCmd, err := c.CmdWrapper(fmt.Sprintf("sudo mv %s %s", tf.Name(), adjustedTempName))
+		mvCmd, err := c.CmdWrapper(fmt.Sprintf("mv %s %s", tf.Name(), adjustedTempName))
 		if err != nil {
 			return err
 		}
 		defer os.Remove(adjustedTempName)
 		ShellCommand(mvCmd).Run()
 		// change cpCmd to use new file name as source
-		cpCmd, err = c.CmdWrapper(fmt.Sprintf("sudo cp %s %s", adjustedTempName, dst))
```

Comment:
I forgot about this part. This wasn't my finest work.
In general we should try to prefer splatting our args to system calls, and I think `CmdWrapper` just wants a string (maybe it splats things?)

`%s` here is also a little dangerous because it allows for arbitrary command execution (if we don't splat the string into an array, when passed to exec/system)

Since this was already here I can live with it, I'll just make a mental note to spruce this up later. Can't imagine why I didn't do this before.